control/server.py

```diff
@@ -91,14 +86,34 @@ def __exit__(self, exc_type, exc_value, traceback):
     def serve(self):
         """Starts gateway server."""
 
+        # Start SPDK
+        self._start_spdk()
+
         # Register service implementation with server
+        omap_state = OmapGatewayState(self.config)
+        local_state = LocalGatewayState()
+        gateway_state = ConsistentGatewayState(self.config, local_state,
+                                               omap_state)
+        self.gateway_rpc = GatewayService(self.config, gateway_state,
+                                          self.spdk_rpc, self.spdk_rpc_client)
         self.server = grpc.server(futures.ThreadPoolExecutor(max_workers=1))
         pb2_grpc.add_GatewayServicer_to_server(self.gateway_rpc, self.server)
 
+        # Add listener port
+        self._add_server_listener()
+
+        # Check for existing NVMeoF target state
+        gateway_state.start_update(self.gateway_rpc_caller)
```

Comment:
start_update is only called on serve once so why not pass `gateway_rpc_caller` when creating the `ConsistentGatewayState` object? It seems that it only supports one gateway_rpc_caller anyway.